backends/platform/sdl/macosx/macosx.cpp

```diff
@@ -261,6 +261,21 @@ Common::String OSystem_MacOSX::getDefaultLogFileName() {
 	return Common::String(prefix) + "/Library/Logs/scummvm.log";
 }
 
+Common::String OSystem_MacOSX::getDefaultIconsPath() {
+	const char *prefix = getenv("HOME");
+	if (prefix == nullptr) {
+		return Common::String();
+	}
+
+	const Common::String appSupportFolder = Common::String(prefix) + "/Library/Application Support/ScummVM";
```

Comment:
I would suggest to add a new function in macosx_wrapper.mm

```obj-c
Common::String getAppSupportPathMacOSX() {
	// See comments in getDesktopPathMacOSX() as we use the same methods
	NSArray *paths = NSSearchPathForDirectoriesInDomains(NSApplicationSupportDirectory, NSUserDomainMask, YES);
	if ([paths count] == 0)
		return Common::String();
	NSString *path = [paths objectAtIndex:0];
	if (path == nil)
		return Common::String();
	return Common::String([path fileSystemRepresentation]) + "/ScummVM";
}
````

Beside using the correct API to get the path, this will also be useful if we finally decide to migrate the savegames there.

Reply:
Done. Thanks for suggesting this!

I also moved the icons into an `Icons/` subdirectory in order to match how it now works with Windows and POSIX.